src/main/java/at/hannibal2/skyhanni/features/dungeon/DungeonSecretChime.kt

```diff
@@ -0,0 +1,49 @@
+package at.hannibal2.skyhanni.features.dungeon
+
+import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.ClickType
+import at.hannibal2.skyhanni.events.BlockClickEvent
+import at.hannibal2.skyhanni.skyhannimodule.SkyHanniModule
+import at.hannibal2.skyhanni.utils.BlockUtils
+import at.hannibal2.skyhanni.utils.BlockUtils.getBlockAt
+import at.hannibal2.skyhanni.utils.SoundUtils
+import at.hannibal2.skyhanni.utils.SoundUtils.playSound
+import net.minecraft.init.Blocks
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent
+
+@SkyHanniModule
+object DungeonSecretChime {
+    private val config get() = SkyHanniMod.feature.dungeon.secretChime
+
+    @SubscribeEvent
+    fun onBlockClick(event: BlockClickEvent) {
+        if (!isEnabled() || event.clickType != ClickType.RIGHT_CLICK) return
+
+        val position = event.position
+        val blockType = when (position.getBlockAt()) {
+            Blocks.chest, Blocks.trapped_chest, Blocks.lever, Blocks.skull -> true
+            else -> false
+        }
```

Comment:
instead of setting a boolean variable here, just return where you are currently setting it to false. If you want to make this code better you would also do the wither essence texture check in this when statement if the block is of skull type